src/skills-builder/skills-builder-modal/view-results/RelatedSkillsSelectableBoxSet.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {
+  SelectableBox, Chip, Stack, useMediaQuery, breakpoints,
+} from '@edx/paragon';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import messages from './messages';
+
+const RelatedSkillsSelectableBoxSet = ({ jobSkillsList, selectedJobTitle, onChange }) => {
+  const { formatMessage } = useIntl();
+  const isExtraSmall = useMediaQuery({ maxWidth: breakpoints.extraSmall.maxWidth });
+
+  return (
+    <SelectableBox.Set
+      name="selected job title"
+      type="radio"
+      value={selectedJobTitle}
+      onChange={onChange}
+      columns={isExtraSmall ? 1 : 3}
+    >
+      {jobSkillsList.map(job => (
+        <SelectableBox
+          key={job.id}
+          type="radio"
+          value={job.name}
+          aria-label={job.name}
+          inputHidden={false}
+        >
+          <p>{job.name}</p>
+          <Stack gap={2} className="align-items-start">
+            <p className="heading-label x-small">{formatMessage(messages.relatedSkillsHeading)}</p>
+            {job.skills.slice(0, 5).map((skill, idx) => (
```

Comment:
Are we just picking/displaying the first 5 skills? Are these selected based on their signifigance? Or some other ranking system?

Reply:
Good catch. This is just taking the top five that are returned, no sorting done. We can use an array method to sort by significance, which I think is probably best?

Reply:
Sounds decent to me! I think this would certainly be a great thing to have for MVP.